### ISSUE TITLE
Button plugin checkbox/radio button lacks visual differentiation between unchecked+focused and checked

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#19742]
Visually, there's no difference (with the exception of the very faint outline) between an unchecked but focused checkbox/radio button and a checked, but not focused, checkbox/radio button. This can be quite confusing.

http://getbootstrap.com/javascript/#buttons-checkbox-radio
http://v4-alpha.getbootstrap.com/components/buttons/#checkbox-and-radio-buttons

> <img width="290" alt="capture2" src="https://cloud.githubusercontent.com/assets/895831/14564897/7bc6a1b6-0320-11e6-852c-0b2de54ff9fc.PNG">
> <img width="368" alt="capture" src="https://cloud.githubusercontent.com/assets/895831/14564898/7bc80290-0320-11e6-8daf-c21dee90a3b2.PNG">

Either the checked or the unchecked+focused state should be changed to provide clearer differentiation. /cc @cvrebert @mdo

**Comment by danger89** [comment]
Maybe take a look at Bootstrap Switch for some inspiration... 
http://www.bootstrap-switch.org/examples.html

**Comment by patrickhlauke** [comment]
it's nice but doesn't actually address the problem of multiple radio buttons next to each other (only provides two hidden radio buttons for on/off)

**Comment by mdo** [comment]
Focus styles were revisited recently, so this should be better now.
